github/PaginatedList.py

```diff
@@ -155,7 +155,12 @@ def totalCount(self):
                 headers=self.__headers
             )
             if 'link' not in headers:
-                self.__totalCount = len(data) if data else 0
+                if data and data.has_key("total_count"):
```

Comment:
```suggestion
                if data and "total_count" in data:
```
`in` operator is better than `has_key`
https://stackoverflow.com/questions/1323410/has-key-or-in

Reply:
Nice catch! I've pushed up that change.